.eslintrc.json

```diff
@@ -1,14 +1,13 @@
 {
   "root": true,
-  "parser": "@typescript-eslint/parser",
```

Comment:
using overrides for various types of files is much easier to maintain, and is the whole premise of https://github.com/NullVoxPopuli/eslint-configs (and how `configs.ember()` can be TS, JS, App, Addon, v2 Addon, all in one)

Reply:
Yeah, this is totally fine with me.